src/VS/FStar.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio 15
+VisualStudioVersion = 15.0.28307.1082

```

Comment:
This is still VS 2017, right?

Reply:
Yes. It's perfectly fine to revert those two lines though. 

I didn't try updating to VS2019 as you suggested I could expect running into some issues by doing that and I was waiting for @tahina-pro to provide more details.